vcbuild.bat

```diff
@@ -126,6 +126,8 @@ if "%target%"=="Clean" rmdir /S /Q %~dp0deps\icu
 
 call :getnodeversion || exit /b 1
 
+if "%target%"=="Clean" rmdir /Q /S "%~dp0%config%\node-v%FULLVERSION%-win-%target_arch%" > nul 2> nul
```

Comment:
`clean` will clear stuff under the selected configuration `config` (which defaults to `Release` unless `debug` is passed as an argument to `vcbuild.bat`).